JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -35,7 +43,54 @@ FlowSensor,Class,https://saref.etsi.org/core/Sensor,IS-A,,,,A device that is use
+Antenna,Class,Accessory,IS-A,,,,A device that is used to transmit or receive signal via electromagnetic radiation.,https://www.theworldavatar.com/kg/ontodevice/,Antenna
+ComputerPart,Class,Accessory,IS-A,,,,A device that is part of or can be added to a computer.,https://www.theworldavatar.com/kg/ontodevice/,Computer Part
+CPU,Class,ComputerPart,IS-A,,,,A device that is responsible for the logical or arithmatic operation in a computer.,https://www.theworldavatar.com/kg/ontodevice/,CPU
+EthernetCard,Class,ComputerPart,IS-A,,,,A device that acts as a network interface betweena a computer and the internet via cable,https://www.theworldavatar.com/kg/ontodevice/,Ethernet card
+Monitor,Class,ComputerPart,IS-A,,,,A device that acts as a display of a computer.,https://www.theworldavatar.com/kg/ontodevice/,Monitor
+Tower,Class,ComputerPart,IS-A,,,,A device that holds computational component and interface of a computer in an enclosed space.,https://www.theworldavatar.com/kg/ontodevice/,Tower
+RAM,Class,ComputerPart,IS-A,,,,A device that provides non-permanent memory during computation to a computer.,https://www.theworldavatar.com/kg/ontodevice/,RAM
+Dock,Class,ComputerPart,IS-A,,,,A device that simplifies a computers cable peripheral interface.,https://www.theworldavatar.com/kg/ontodevice/,Dock
+Printer,Class,ComputerPart,IS-A,,,,A device that prints data or documents to a paper.,https://www.theworldavatar.com/kg/ontodevice/,Printer
+SSD,Class,ComputerPart,IS-A,,,,A device that provies permanent memory on a computer.,https://www.theworldavatar.com/kg/ontodevice/,SSD
+PCCooler,Class,ComputerPart,IS-A,,,,A device that lowers the operating temperature of a computer during operation.,https://www.theworldavatar.com/kg/ontodevice/,PC Cooler
+Screen,Class,ComputerPart,IS-A,,,,A device on which a computer interface is projected onto.,https://www.theworldavatar.com/kg/ontodevice/,Screen
+Projector,Class,ComputerPart,IS-A,,,,A device tht projects an image to a screen,https://www.theworldavatar.com/kg/ontodevice/,Projector
+Speaker,Class,ComputerPart,IS-A,,,,A device that converts electrical signal from  acomputer to soundwaves.,https://www.theworldavatar.com/kg/ontodevice/,Speaker
+Scanner,Class,ComputerPart,IS-A,,,,A device that  scans paper-based documents to a digital format,https://www.theworldavatar.com/kg/ontodevice/,Scanner
+Dongle,Class,ComputerPart,IS-A,,,,"A small piece of computer hardware that connects to a port on another device to provide it with additional functionality, or enable a pass-through to such a device that adds functionality.",https://www.theworldavatar.com/kg/ontodevice/,Dongle
+ServerChassis,Class,ComputerPart,IS-A,,,,A device where servers are mounted on.,https://www.theworldavatar.com/kg/ontodevice/,Server chassis
+NetworkVideoRecorder,Class,ComputerPart,IS-A,,,,A device that converts video recording in a camera network to a computer-storable format,https://www.theworldavatar.com/kg/ontodevice/,Network video recorder
```

Comment:
Not sure whether to classify a network video recorder as a computer part? Because it can also upload data to other storage devices like usb, sd card etc and does not necessarily need to be connected to a computer.

In addition, if we take into account other recording devices (e.g. video cameras), we can't slot these devices under computer part.